service/testdata/TestCreateTarget/fails_to_create_target_when_bad_auth_header_response.json

```diff
@@ -0,0 +1,3 @@
+{
+  "error_message": "error unauthorized, invalid authorization header format"
```

Comment:
Why the different messages between this and [here](https://github.com/argoproj-labs/argo-cloudops/pull/270/files#diff-254971a6c4ca6008a056da66724fa70f6497da16bafadeacc80f4fad087842d1R2)?